roles/forklift_versions/molecule/default/verify.yml

```diff
@@ -59,7 +45,7 @@
       vars:
         scenario: "{{ pipeline_type }}"
         scenario_os: "{{ pipeline_os }}"
-        scenario_version: "4.9"
+        scenario_version: "4.8"
```

Comment:
Same issue as above, Katello 4.8 doesn't exist on EL9
You can set it to 4.13 and then assert that start is 4.12, intermediate is 4.12 (as we didn't have anything before that on el9) and final is 4.13.